api/draft_registrations/serializers.py

```diff
@@ -46,6 +48,13 @@ class DraftRegistrationSerializer(DraftRegistrationLegacySerializer, Taxonomizab
     tags = ValuesListField(attr_name='name', child=ser.CharField(), required=False)
     node_license = NodeLicenseSerializer(required=False, source='license')
 
+    registration_schema = RegistrationSchemaRelationshipField(
+        related_view='schemas:registration-schema-detail',
+        related_view_kwargs={'schema_id': '<registration_schema._id>'},
+        required=True,
+        read_only=False,
+    )
```

Comment:
Notes pending CI verification: I don't think this is necessary since there is no conflicts between `DraftRegistrationLegacySerializer` and `TaxonomizableSerializerMixin`. `DraftRegistrationLegacySerializer` defines the `registration_schema` as it is used/redefined here.

Reply:
Confirmed that we don't need this one (it doesn't add or remove CI failures/errors)